.gitmodules

```diff
@@ -91,4 +91,4 @@
 [submodule "pmm-qa"]
 	path = sources/pmm-qa/src/github.com/percona/pmm-qa
 	url = https://github.com/percona/pmm-qa
-	branch = master
+	branch = update-version-to-2.7.1
```

Comment:
@puneet0191 please revert it ones you merge your PR and rewind submodules

Reply:
okay thanks